internal/cli/util/flags/flags.go

```diff
@@ -44,41 +43,15 @@ func AddClusterDefinitionFlag(f cmdutil.Factory, cmd *cobra.Command, p *string)
 		}))
 }
 
-func AddComponentsFlag(f cmdutil.Factory, cmd *cobra.Command, isPlural bool, p any, usage string) {
```

Comment:
Recommended to split into two functions and clarify their purpose through naming.
`AddComponentsFlag` and `AddComponentFlag`, remove the `isPlural` parameter.
 
And, may be move `cluster.GetClusterByName` to util package, instead of move `AddComponentsFlag` to `cluster` package.

Reply:
I'm using a hack approach where I've rewritten the logic for GetClusterByName in flags.go to resolve the circular import issue. Moving cluster.GetClusterByName would be a significant change for the entire project.